kitsune/community/templates/community/email/first_answer.ltxt

```diff
@@ -0,0 +1,58 @@
+{# This is an email.  Whitespace matters! #}
+{% autoescape false %}
+{% trans %}
+Thank you for your first contribution to the Mozilla support forums! You just
+made somebody's day!
+{% endtrans %}
+
+{% trans %}
+Mozilla support is fully powered by our volunteer community that is helping
+hundreds of users everyday. This community is formed by passionate Firefox and
+Thunderbird users who take some of their spare time to help others. We couldn't
+do it without your passion and enthusiasm. Your contribution, as minor as it
+might seem, has tremendous value. Solving one issue in the support forums can
+help up to 1000 users a day!
+{% endtrans %}
+
+---
+{{ _('Tell us about yourself!') }}
+
+{% trans %}
+We are always very happy to hear from new people who like helping on the forums
+so If you haven’t met our community yet please take a minute to introduce
+yourself. If you want to stay low-profile, that’s also ok :). You can also find
+us all hanging out on IRC, on the #sumo channel on irc.mozilla.org.
+{% endtrans %}
+
+{% trans url='https://support.mozilla.org/en-US/forums/contributors' %}
+The community forum is at {{ url }}.
+{% endtrans %}
+
+---
```

Comment:
Oh, interesting.

Maybe:

```
NEW SECTION!
```

?

Kidding. If we do this elsewhere and it was an issue, someone (hopefully) would have filed an issue by now.